package.json

```diff
@@ -80,7 +80,7 @@
     "cssom": "^0.4.4",
     "del": "^2.2.2",
     "devextreme-cldr-data": "^1.0.2",
-    "devextreme-internal-tools": "^7",
+    "devextreme-internal-tools": "^7.7.0",
```

Comment:
Hm... it seems that we agreed to keep "^7" version before.
Better leave it as is then, and update only package-lock

Reply:
In that case - shouldn't I also revert this change in **package-lock.json**
```diff
-        "devextreme-internal-tools": "^7",
+        "devextreme-internal-tools": "^7.7.0",
```
and keep only that:

```diff
 "node_modules/devextreme-internal-tools": {
-      "version": "7.6.0",
+      "version": "7.7.0",
```

Reply:
I guess, yes